### PR TITLE
[new release] git, git-cohttp, git-cohttp-unix, git-cohttp-mirage, git-unix and git-mirage (3.1.1)

### PR DESCRIPTION
**Pull request by dinosaure**
Git format and protocol in pure OCaml

- Project page: <a href="https://github.com/mirage/ocaml-git">https://github.com/mirage/ocaml-git</a>
- Documentation: <a href="https://mirage.github.io/ocaml-git/">https://mirage.github.io/ocaml-git/</a>

##### CHANGES:

- Fix `git-unix` and PACK files location (@dinosaure, mirage/ocaml-git#444, mirage/ocaml-git#443)
- Initialise (as `git`) correctly (add an `HEAD` reference, at least) (@dinosaure, mirage/ocaml-git#443)

### CODE REVIEW

File: packages/git-cohttp-mirage/git-cohttp-mirage.3.1.1/opam [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  "alcotest-lwt" {>= "1.2.3" & with-test}
+  "bigstringaf" {>= "0.7.0" & with-test}
+  "cstruct" {>= "6.0.0" & with-test}
+  "logs" {>= "0.7.0" & with-test}
+  "mirage-flow" {>= "2.0.1" & with-test}
+  "bigarray-compat" {>= "1.0.0" & with-test}
+  "ke" {>= "0.4" & with-test}
+]
+build: [
+  ["dune" "subst"]
```
Comment (kit-ty-kate) [inline packages/git-cohttp-mirage/git-cohttp-mirage.3.1.1/opam:32]: Is there a reason why `dune subst` is called unconditionally?

Comment (dinosaure) [reply packages/git-cohttp-mirage/git-cohttp-mirage.3.1.1/opam:32]: Should I add `{pinned}` or remove it?

Comment (kit-ty-kate) [reply packages/git-cohttp-mirage/git-cohttp-mirage.3.1.1/opam:32]: If you're using substitutions then adding `{dev}` is the way to go. If not, either way is ok.

Side note: using `{pinned}` in this case was a bug and should not be used

Comment (dinosaure) [reply packages/git-cohttp-mirage/git-cohttp-mirage.3.1.1/opam]: So, I removed all `dune subst` :+1:.

**Comment by kit-ty-kate** [comment]
Thanks a lot!

**Pull request merged**